pkgs/dartpad_ui/lib/console.dart

```diff
@@ -118,6 +119,61 @@ class _ConsoleWidgetState extends State<ConsoleWidget> {
                   ],
                 ),
               ),
+              // If there is a JS error, and the app type is Flutter, show a
+              // "File an issue" button. JS errors are normal when executing
+              // Dart (non-Flutter) code.
+              if (widget.output.hasError &&
+                  widget.output.hasJavascriptError &&
+                  appModel.appIsFlutter.value == true)
+                Align(
+                  alignment: Alignment.bottomRight,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    children: [
+                      Text(
+                        'Oops! Something went wrong.',
```

Comment:
I'm not sure how much space we have here. If not much, this verbiage looks fine. If a little more, we might want to explain that the problem could have been related to the DDC compiler ('This might have been caused by a compilation error with the Dart toolchain. Report an issue?').